uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
+    <table class="table">
+      <tr>
+        <td>
+          <strong>Position</strong>
+        </td>
+        <td>
```

Comment:
you should add some widths to the `td`s otherwise the width will change based on data. But it should be fixed. Also the EDIT ENTITLEMENTS section should be smaller as it is in the mockup. Adding width will fix this too.